cnd/src/config/mod.rs

```diff
@@ -28,11 +30,37 @@ pub struct Socket {
 pub struct Bitcoin {
     #[serde(with = "crate::config::serde_bitcoin_network")]
     pub network: bitcoin::Network,
+    pub bitcoind: Bitcoind,
```

Comment:
This structure doesn't represent your config. I think the `connectors` is missing here.

Reply:
I opted for removing the `connectors`  thingy if you look at the example above (see your comment https://github.com/comit-network/comit-rs/pull/2053/files#r381681455)

Reply:
Ah okay, I thought I still saw it somewhere. The reason I added it in my proposal in the ticket is because `bitcoin.bitcoind` reads a bit weird IMO. But I am fine either way, less noisy also has its advantages.